src/features/dataset/state/datasetActions.ts

```diff
@@ -190,6 +194,54 @@ export interface ResetDatasetStateAction {
   type: string
 }
 
+export interface SetDatasetMetaAction {
+  type: string
+  meta: Meta
+}
+
+export interface SetDatasetReadmeAction {
+  type: string
+  readme: Readme
+}
+
+export interface SetDatasetEditableAction {
+  type: string
+  editable: boolean
+}
+
+export interface SetDatasetCommitTitleAction {
+  type: string
+  title: string
+}
+
+export function setDatasetCommitTitle (title: string): SetDatasetCommitTitleAction {
+  return {
+    type: SET_DATASET_COMMIT_TITLE,
+    title
+  }
+}
+
+export function setDatasetEditable (editable: boolean): SetDatasetEditableAction {
+  return {
+    type: SET_DATASET_IS_EDITABLE,
+    editable
+  }
+}
+
+export function setDatasetMeta (meta: Meta): SetDatasetMetaAction {
+  return {
+    type: SET_DATASET_META,
+    meta
+  }
+}
+
+export function setDatasetReadme (readme: Readme): SetDatasetReadmeAction {
+  return {
+    type: SET_DATASET_README,
+    readme
+  }
+}
+
```

Comment:
It looks like most of these may have been replaced by the action in `src/features/datasetEditor/state/datasetEditorActions`

Unless they are being saved for future purposes, we should remove them.